db/migrate/20170728004206_update_terms_and_conditions_yes_content.rb

```diff
@@ -0,0 +1,15 @@
+class UpdateTermsAndConditionsYesContent < ActiveRecord::Migration
+  def up
+    yes_content = 'Yes, I agree with the terms and conditions for medical information'
+    terms_and_conditions = TermsAndConditions.find_by(version: '1.0')
```

Comment:
Taking a look at https://github.com/department-of-veterans-affairs/vets-api/blob/master/db/migrate/20170519153047_create_mhv_ac_terms.rb#L49

I think a more reliable lookup is going to be (there is probably only the one record, but we can't know that for sure):

`TermsAndConditions.find_by(name: 'mhvac', version: '1.0')`